x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -217,9 +218,12 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
   }
 
   public start(core: CoreStart, plugins: StartPlugins) {
+    plugins.ingestManager.register('datasourceCreate', handleDatasourceCreate);
```

Comment:
we have a plugin test where we verify the calls that are made. And we have the context service start, might consider moving it in there.